packages/react/src/wallet/useWalletLogin.ts

```diff
@@ -16,7 +16,7 @@ export function useWalletLogin() {
   const loginWallet = useWalletLoginController();
 
   return {
-    login: async (signer: Signer) => {
+    login: async (signer: Signer, profileId?: string) => {
```

Comment:
Wondering... in general knowing your own profile Id is very unlikely. Should we use the profile handle instead?

Reply:
Perhaps but the usecase I was thinking for this functionality would be querying the api and displaying a list of profiles to log into. Should we support handle in a separate task?